packages/datetime/src/dateRangeInput.tsx

```diff
@@ -512,6 +753,10 @@ export class DateRangeInput extends AbstractComponent<IDateRangeInputProps, IDat
         return (boundary === DateRangeBoundary.START) ? DateRangeBoundary.END : DateRangeBoundary.START;
     }
 
+    private areSameDay = (a: moment.Moment, b: moment.Moment) => {
```

Comment:
Didn't use `DateUtils.areSameDay`, because that function expects`Date` parameters, not `moment.Moment`s. Don't love that this new function lives here, but given that it's not used anywhere else, I thought it was okay for now.